aces/Loris-MRI
# Fixing logic for excluded series from the dcm2mnc conversion

This fixes a bug in the logic of the excluded series from the dcm2mnc conversion.

Make sure to test those three situations:
1) run the pipeline with 2 or more excluded series in the Config module
2) run the pipeline with only 1 excluded series in the Config module
3) run the pipeline with no excluded series in the Config module

See also: https://redmine.cbrain.mcgill.ca/issues/14916
